src/Transition/CreateNewIDFUsingRulesV8_6_0.f90

```diff
@@ -728,6 +727,36 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
                   OutArgs(1)='*'
                   nodiff=.false.
                 ENDIF
+
+                ! For Output:Variable that reference a specific Daylighting:Controls object need update to the new reference point name
+                IF (InArgs(1) .NE. '*') THEN
+                  isDElightOutVar = .FALSE. ! first assume that it is not a DElight related variable
+                  IF (SameString(InArgs(2)(1:27),'Daylighting Reference Point')) THEN
```

Comment:
I think part of what is confusing is due to a subtle design decision that I made to try to make it as seamless as possible for current SplitFlux and DElight users:

- For SplitFlux the name of the Daylighting:Controls object is used as the key for all of "reference point" related output variables. That is why they need to be numbered 1, 2, 3... This is how it was in 8.5.0
- For DElight the name of the Daylighting:ReferencePoint is used as the key for all the "reference point" related output variables. Since the names are unique they are not numbered. This is how it was in 8.5.0

If we are going to redo things, I would say we should drop the numbering and change the key names for the SplitFlux output variables to be the name of the Daylighting:ReferencePoint objects. They would be unique and this would be consistent between DElight and SplitFlux. 

Unfortunately, this is probably a little more work and a little riskier given the timeline we are on for the release. It means changing stuff in EnergyPlus and Transition.  Please let me know if you want me to proceed.